pkg/daemon/rpm-ostree.go

```diff
@@ -100,6 +115,60 @@ func (r *RpmOstreeClient) loadStatus() (*rpmOstreeState, error) {
 	return &rosState, nil
 }
 
+func parseVer(s string) ([]int, error) {
+	r := []int{}
+	for _, s := range strings.Split(s, ".") {
+		n, err := strconv.Atoi(s)
+		if err != nil {
+			return nil, fmt.Errorf("Failed to parse %s: %v", s, err)
+		}
+		r = append(r, n)
+	}
+	return r, nil
+}
+
+func validateVersion(current rpmOstreeVersionOuter) error {
```

Comment:
```suggestion
func validateRpmOstreeVersion(current rpmOstreeVersionOuter) error {
```

clearer and in line with below function